code/modules/events/anomaly/anomaly_ectoplasm.dm

```diff
@@ -0,0 +1,64 @@
+/datum/round_event_control/anomaly/anomaly_ectoplasm
+	name = "Anomaly: Ectoplasmic Outburst"
+	description = "Anomaly that produces an effect of varying intensity based on how many ghosts are orbiting it."
+	typepath = /datum/round_event/anomaly/anomaly_ectoplasm
+	min_players = 30
+	max_occurrences = 2
+	weight = 4 //Rare because of it's wacky and silly nature
+	category = EVENT_CATEGORY_ANOMALIES
+	admin_setup = /datum/event_admin_setup/anomaly/anomaly_ectoplasm
+
+/datum/round_event/anomaly/anomaly_ectoplasm
+	anomaly_path = /obj/effect/anomaly/ectoplasm
+	start_when = 3
+	announce_when = 20
+	///The admin-set impact effect intensity override
+	var/effect_override
+	///The admin-set number of ghosts, for use in calculating impact size.
+	var/orbit_override
+
+/datum/round_event/anomaly/anomaly_ectoplasm/start()
+	if(!effect_override)
+		..() //If we provide no override, just run the usual startup.
+	else
+		var/turf/anomaly_turf = placer.findValidTurf(impact_area)
+		var/obj/effect/anomaly/ectoplasm/newAnomaly
+		if(anomaly_turf)
+			newAnomaly = new anomaly_path(anomaly_turf)
+			newAnomaly.override_ghosts = TRUE
+			newAnomaly.effect_power = effect_override
+			newAnomaly.ghosts_orbiting = orbit_override
+			newAnomaly.intensity_update()
+		if (newAnomaly)
+			announce_to_ghosts(newAnomaly)
+
+/datum/round_event/anomaly/anomaly_ectoplasm/announce(fake)
+	priority_announce("Localized ectoplasmic outburst detected on long range scanners. Expected location of impact: [impact_area.name].", "Anomaly Alert")
+
+/datum/event_admin_setup/anomaly/anomaly_ectoplasm
+	var/ghost_override
+	var/chosen_effect
+
+/datum/event_admin_setup/anomaly/anomaly_ectoplasm/prompt_admins()
+	..()
+
+	if(tgui_alert(usr, "Override the anomaly effect and power?", "You'll be ruining the authenticity.", list("Yes", "No")) == "Yes")
+		var/list/power_values = list("Minor", "Moderate", "Major")
+		chosen_effect = tgui_input_list(usr, "Provide effect override", "Criiiiinge.", power_values)
+		ghost_override = tgui_input_number(usr, "How many ghosts do you want simulate orbiting your anomaly? (determines the effect radius).", "Seriously, CRINGE.", 0, 20, 1)
+
+		if(!chosen_effect || !ghost_override)
+			return ADMIN_CANCEL_EVENT
```

Comment:
If you're going to bully the admin, you should at least have it return ADMIN_CANCEL_EVENT after each prompt like

tgui_alert(chosen_values)
if(!chosen_values)
    return
tgui_alert(ghost_override)
    return

just to be clear that when they exit out, it actually cancels the event rather than fill in for `chosen_effect` or something and not fail silently.